msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -547,6 +694,7 @@ The object is an array (maximum of 1 element) with all elements of type `object`
 
 |Name| Type | Maximum Size | Required | Description|
 |---|---|---|---|---|
+|`id`|String||| Unique identifier for compose extension. Used when defining one-way and mutual app component dependencies as [elementRelationshipSet](#elementrelationshipset).|
```

Comment:
> Unique identifier for compose extension.

@erikadoyle, do you mean `composeExtension` or message extension?

Reply:
Seems like the convention here is to say "message extension". Updated.

Reply:
Agreed, @erikadoyle. Let's stick with message extension.